### ISSUE TITLE
Layout for the default widget for FileField looks broken

### ISSUE THREAD

**Issue by m1kola** [wagtail/wagtail#4780]
### Issue Summary

Layout for the default widget for FileField looks broken. The template seems to come from Django itself.

### Steps to Reproduce

1. Create a page with a `models.FileField`. For example:
    ```python
    header_video = models.FileField(blank=True, verbose_name='Hero video')
    ```
2. Add a `FieldPanel` into `content_panels`:
    ```python
    FieldPanel('header_video')
    ````

**Result:**

![filefield](https://user-images.githubusercontent.com/509198/45698499-10eca980-bb60-11e8-93ff-af900d5f595d.png)

### Technical details

* Python version: 3.6.6
* Django version: 2.0.8
* Wagtail version: 2.2.2
* Browser version: Google Chrome 69.0.3497.81 and FireFox 63.0b7s and Safari 11.0.1

**Comment by m1kola** [comment]
I think, it happens because `FieldPanel` generates a form using Django's [`ClearableFileInput`](https://docs.djangoproject.com/en/2.0/ref/forms/widgets/#django.forms.ClearableFileInput) widget and Wagtail doesn't provide styles for it.

**Comment by nmorduch** [comment]
I'm only getting the file input, with no checkbox for clearing it. It's still in a div with `.clearable_file_input` so I'm really not sure why the checkbox input isn't there.

<img width="1030" alt="image" src="https://user-images.githubusercontent.com/1051450/48242106-a873b900-e3a7-11e8-93f1-cd484237ede0.png">

**Comment by lb-** [comment]
Fixed in #4790 via https://github.com/wagtail/wagtail/commit/d2d8b2c9e343e6a9532d615f079dd13e7003e80e
